src/modules/core/mail-server/mailServer.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+const handlebars = require("nodemailer-express-handlebars");
+const nodemailer = require("nodemailer");
+
+require("dotenv").config();
+const { GMAIL_ADDRESS, GMAIL_PASSWORD } = process.env;
+
+function sendMail(receiver) {
+	const transporter = nodemailer.createTransport({
```

Comment:
keep the transporter out of the sendMail function. As it is independent logic. so keep this in global scope

Reply:
Updated